SabreTools.Serialization/Wrappers/ISO9660.Extraction.cs

```diff
@@ -125,6 +128,9 @@ private bool ExtractFile(DirectoryRecord dr, Dictionary<int, int> extractedFiles
             const int chunkSize = 2048 * 1024;
             lock (_dataSourceLock)
             {
+                // Skip the blocks allocated to the Extended Attribute Record
+                fileOffset += dr.ExtendedAttributeRecordLength * blockLength;

```

Comment:
It seems reasonable to move this to directly below the current place where `fileOffset` is set.